sources/academy/webscraping/scraping_basics_javascript2/05_parsing_html.md

```diff
@@ -38,6 +38,20 @@ $ pip install beautifulsoup4
 Successfully installed beautifulsoup4-4.0.0 soupsieve-0.0
 ```
 
+<!--
```

Comment:
This is removed and addressed in subsequent commits of https://github.com/apify/apify-docs/pull/1584, which are yet about to become Pull Requests.